docs/resources/lakeview_dashboard.md

```diff
@@ -0,0 +1,54 @@
+---
+subcategory: "Workspace"
+---
+# databricks_lakeview_dashboard Resource
+
+This resource allows you to manage [Databricks Lakeview Dashboards](https://docs.databricks.com/api/workspace/lakeview). To manage [Lakeview Dashboards](https://docs.databricks.com/api/workspace/lakeview) you must have a warehouse access on your databricks workspace.
```

Comment:
Nit, but from the user reading the final doc, it feels like [Databricks Lakeview Dashboards] and [Lakeview Dashboards] are two different things/links.

Reply:
Moved Databricks out of the link